### PR TITLE
Add elixir lang support

### PR DESCRIPTION
**Pull request by samontea**
Elixir is a pretty cool language. We should have support for it by default. 

⚗⚗⚗

### CODE REVIEW

File: .emacs.d/lisp/package/dependencies.el [hunk 1]
```diff
@@ -17,6 +17,7 @@
     hl-todo ;; highlight TODO, FIXME, FAIL, etc.
     highlight-symbol ;; navigate between and highlight symbols
     web-mode ;; for editing web files
+    elixir-mode
```
Comment (rye) [inline .emacs.d/lisp/package/dependencies.el]: Put a comment here (e.g. `;; for editing Elixir code`)

File: .emacs.d/lisp/code/languages/elixir.el [hunk 2]
```diff
@@ -0,0 +1,3 @@
+(require 'elixir-mode)
+
+(provide 'elixir-configuration)
```
Comment (rye) [inline .emacs.d/lisp/code/languages/elixir.el]: Since your file is named `elixir`, you should provide `'elixir`.

**Review by rye — CHANGES_REQUESTED** [review]
Almost there, but you also need to add `elixir` to the list in `language-hub.el` to have the system actually load this.

File: .emacs.d/lisp/code/languages/language-hub.el [hunk 3]
```diff
@@ -1,5 +1,6 @@
 (defvar languages
   '(c
+    elixir
```
Comment (rye) [inline .emacs.d/lisp/code/languages/language-hub.el]: Alphabetize?

**Review by rye — APPROVED** [review]

**Review by rye — APPROVED** [review]
LGTM.

**Comment by rye** [comment]
Since @cg505 is probably busy because he's overseas at the moment, I'll go ahead and merge this lil' guy. Thanks, @samontea! 🎉

**Comment by cg505** [comment]
ty @rye

**Pull request merged**